WordPressUIKit/WordPressUIKit/Categories/UIImage+Alpha.h

```diff
@@ -1,13 +0,0 @@
-// UIImage+Alpha.h
```

Comment:
Can we really nuke this when it's used by `roundedCornerImage`?

Reply:
I've nuked that as well. Not used anywhere in the project.

🔥